cmd/vclusterctl/cmd/platform/backup/management.go

```diff
@@ -67,8 +68,10 @@ vcluster platform backup management
 		},
 	}
 
+	platformNamespace, _ := clihelper.VClusterPlatformInstallationNamespace(context.Background())
```

Comment:
Is it safe to ignore this error here?

Reply:
Yeah, its a bit weird.  If this fails the namespace flag value will just be an empty string and [this prompt](https://github.com/loft-sh/vcluster/blob/main/cmd/vclusterctl/cmd/platform/backup/management.go#L96-L99) will allow them to proceed.  The backup will proceed, but with warnings and output like

```shell
09:52:23 info Backing up clusterroletemplates...
09:52:23 info Backing up clusteraccesses...
...
09:52:23 info Backing up runners...
09:52:23 info Backing up projects...
09:52:23 warn backup clusterrole templates: no matches for kind "ClusterRoleTemplate" in version "storage.loft.sh/v1"
09:52:23 warn backup cluster accesses: no matches for kind "ClusterAccess" in version "storage.loft.sh/v1"
09:52:23 warn backup users: no matches for kind "User" in version "storage.loft.sh/v1"
09:52:23 warn backup teams: no matches for kind "Team" in version "storage.loft.sh/v1"
...
09:52:23 warn backup runners: no matches for kind "Runner" in version "storage.loft.sh/v1"
09:52:23 warn backup projects: no matches for kind "Project" in version "storage.loft.sh/v1"
```

So in the interests of having it work like it did before, I just ignored the error, and I didn't fall back to `vcluster-platform` since that should have been found if it was a valid alternative.